docs/Users_Guide/release-notes.rst

```diff
@@ -27,7 +27,7 @@ Version `v6.2.0 rc1 <https://github.com/dtcenter/METviewer>`_ Release Notes (202
 
   .. dropdown:: Bugfixes
 
-     *Bugfix- METviewer6.1 batch script no longer works (`#605 <https://github.com/dtcenter/METviewer/issues/605>`_)
+     * Bugfix- METviewer6.1 batch script no longer works (`#605 <https://github.com/dtcenter/METviewer/issues/605>`_)
```

Comment:
I'd suggest turning the language around a bit. For example, instead of saying "Bugfix- METviewer6.1 batch script no longer works", I'd say something like "Fix METviewer6.1 batch script", but that's just me being nit picky. :) 
Feel free to change or not. Thanks for catching the space issue!